src/api/server.tsx

```diff
@@ -1,18 +1,20 @@
 import React, { useEffect, useState } from "react";
 
+const BASE_URL = process.env.REACT_APP_APIURL;
 enum Url {
-  GetList = "http://127.0.0.1:8000/todo-list/",
-  AddTodo = "http://127.0.0.1:8000/todo",
+  GetList = `full-todo-list`,
+  AddTodo = "full-todo",
+  DeleteTodo = "todo-delete/",
 }
 
 export async function fetchTodoList() {
-  const response = await fetch(Url.GetList);
+  const response = await fetch(BASE_URL + Url.GetList);
```

Comment:
Instead of concatenation you can use string interpolation here like: `${BASE_URL}${Url.GetList}`. 

It seems minor but when. you use `+` the app can crash if any of the parameters are null or undefined.

Reply:
Agreed, but I was getting errors when I tried using string interpolation. Not sure if I'm doing it the right way but I tried doing it your way and received this error:  
<img width="1792" alt="Screen Shot 2023-02-07 at 10 35 18 AM" src="https://user-images.githubusercontent.com/90336648/217290422-abe3668a-d272-4661-b431-db5dfd518dcb.png">

Reply:
You need backticks around the string - so it's 

```
`${BASE_URL|${Url.GetList}`
```

But the backticks are needed for interpolated strings.

Reply:
Agreed. Got it to work!!